src/components/MainTabs/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import BugzillaComponents from '../BugzillaComponents';
+import Reportees from '../Reportees';
+
+const TabContainer = (props) => {
+  const { children } = props;
+  return (
+    <Typography component="div" style={{ padding: 4 }}>
+      {children}
+    </Typography>
+  );
+};
+
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
```

Comment:
Depending on what you pass down this component it will be one of two:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/f834d41a7818dbf80123795b4d57aa6151793eb8/src/quantum/widget.jsx#L129-L132